### PR TITLE
iOS Support

### PR DESCRIPTION
**Pull request by Perksey**
# Summary of the PR
Basic iOS support using SDL

Works on simulator and device.

~~**BLOCKED BY PENDING `ExcludeFromOverride` ATTRIBUTE CHANGES**~~

# Related issues, Discord discussions, or proposals
#48 

# Further Comments

### CODE REVIEW

**Comment by HurricanKai** [comment]
You don't actually want to merge this into master, right?

**Comment by Perksey** [comment]
Oh god no 😂 feel free to retarget it I’ve switched off for tonight as
tomorrow all I’ve gotta do is build it

File: src/Core/Silk.NET.Core/Silk.NET.Core.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```
Comment (HurricanKai) [inline src/Core/Silk.NET.Core/Silk.NET.Core.csproj:4]: Not sure what's going on here, I don't think we properly handle netstandard2.1 when loading libraries yet.

**Review by HurricanKai — COMMENTED** [review]
Looks good!

File: src/Core/Silk.NET.Core/Silk.NET.Core.csproj (hunk 1 above)
Comment (Perksey) [reply src/Core/Silk.NET.Core/Silk.NET.Core.csproj:4]: I don't think we have any TFM-specific loading code, we generally use the deps.json if available.

**Review by HurricanKai — APPROVED** [review]

**Comment by Perksey** [comment]
Ready to ship as "super duper experimental", merging...

Will keep investigating some nasty issues and hope to be confident by preview 4.

**Pull request merged**